### ISSUE TITLE
newrelic/cli:v0.13 events post stoped working

### ISSUE THREAD

**Issue by msporleder** [newrelic/newrelic-cli#385]
### Description
newrelic/cli:v0.13 (official docker image) does not insert events

### Go Version
check your Dockerfile

### Current behavior
v0.13
```
docker run -e NEW_RELIC_INSIGHTS_INSERT_KEY -e NEW_RELIC_API_KEY -eNEW_RELIC_REGION=US --rm newrelic/cli:v0.13 events post --accountId 12345 --event '{ "eventType":"my_custom_thing", "cnt": 9797 }'
time="2020-09-28T15:22:43Z" level=info msg=success
```
go check in insights ... nothing!

### Expected behavior
do the above, except with newrelic/cli:v0.12

go check in insights .. data!

**Comment by zlesnr** [comment]
Thank you for the report @msporleder.  When was the last time this worked for you?

**Comment by msporleder** [comment]
with v0.12. I just tried v0.13 yesterday.

**Comment by zlesnr** [comment]
I was able to reproduce this and took a little spelunking.  The fix in the client should help out here.  We'll need to release the client, bump the version here, and then release the cli.

**Comment by ctrombley** [comment]
Fixed in v0.14.
